cluster/gce/config-test.sh

```diff
@@ -191,6 +191,12 @@ TEST_CLUSTER_RESYNC_PERIOD="${TEST_CLUSTER_RESYNC_PERIOD:---min-resync-period=3m
 TEST_CLUSTER_API_CONTENT_TYPE="${TEST_CLUSTER_API_CONTENT_TYPE:-}"
```

Comment:
yeah, just in testing.  We wont turn it on by default in prod ever, as it can interact poorly with some kernel configs.  We will only enable it in GKE, not GCE